docs/30DaysOfPWA/advanced-capabilities/04.md

```diff
@@ -4,12 +4,148 @@
-| **Exercise** | Test different display modes. Play with the Window Controls Overlay feature. |
-| **Related** | Look for other OS integrations throughout [Week 2: Advanced Capabilities](../advanced-capabilities). |
+Welcome to day 11 of the [30 Days of PWA](<https://aka.ms/learn-pwa/30Days-blog>) series! Today’s blog post will walk you through the different display modes you can set for your website, and the experimental Window Controls Overlay feature which allows the customization of the title bar of your PWA.
+
+## Display Modes 
+There are four different display modes available to choose from and each provides a different browser UI experience that you’ll want to consider when deciding which display mode is right for your progressive web application. 
+
+You set your preferred display mode via the web app manifest’s display property with one of the following options: `fullscreen`, `standalone`, `minimal-ui`, or `browser`:
+
+```
+{
+
+“display”: “standalone”
+
+}
+```
+
+In the event a browser doesn’t support a display mode, which is possible since they’re not required or support all the display mode options, the display mode will fallback through the following display modes this order: `fullscreen`, `standalone`, `minimal-ui`, and `browser`. 
+
+### Standalone display mode 
+Standalone display mode that makes your PWA look and behave the most like a native application. It opens in a different window from the browser and hides all the browser UI elements like the address bar but keeps standard system UI elements like a back button or a close window button. In this mode, your application can also have its own icon in the application launcher.  If standalone isn’t available, it will fall back to the minimal-ui display mode. Twitter uses `standalone` for their PWA.
+
+![Twitter displayed in its own window on Android](_media/display-standalone.png)
+
+### Fullscreen display mode
+Fullscreen display mode takes up the entirety of the display area available and hides all of the browser UI elements. If fullscreen mode isn’t available, it will fall back to standalone. Here's an example of a game that uses `fullscreen`.
+
+![This PacMan game is a PWA and was built with the fullscreen display mode](_media/display-fullscreen.png)
+
+### Minimal-UI display mode 
+The minimal UI display mode gives your PWA a similar experience to the standalone display mode. It opens in its own window, but the application retains a minimal set of the browser UI, and the UI that is retained varies between browsers. The OneDrive PWA uses the `minimal-ui` display mode.
+
+![The OneDrive PWA utilizes the minimal-ui display mode](_media/display-minimal-ui.png) 
+
+### Browser display mode
+The browser display mode retains the browser experience with all the browser UI and your web app will not be installable. It will open in a normal browser window or tab. If Twitter were not using standalone mode, it would display in the browser like a normal tab. 
```

Comment:
```suggestion
The "browser" display mode retains the full browser experience, with all the browser UI users expect. If you use this display mode, browsers will not prompt users to install your web app. And even if it's installed manually, your app will always open in a new browser window or tab. If Twitter were not using "standalone" mode, it would display in the browser like a normal tab.
```